compose/cli/main.py

```diff
@@ -545,12 +545,14 @@ def up(self, project, options):
             --no-recreate          If containers already exist, don't recreate them.
                                    Incompatible with --force-recreate.
             --no-build             Don't build an image, even if it's missing
+            --cascade-stop         Stops all containers if any container was stopped.
```

Comment:
I'm not sure about this name for the flag, I don't think it really captures the behaviour correctly. Let's try and brainstorm some other options.

I'll throw in a couple options:
* `--shutdown-on-first-container-exit` (feels a little too verbose, but I think captures the behaviour well)
* `--shutdown-on-exit` (more concise, less clear)
* `--stop-on-exit`